src/credentials/BUILD.gn

```diff
@@ -79,6 +79,10 @@ static_library("credentials") {
     ]
   }
 
+  if (chip_enable_trustm_da == true) {
+    sources += [ "${chip_root}/examples/platform/infineon/trustm/DeviceAttestationCredsExampleTrustM.cpp"]
```

Comment:
This will include an example. What about developers trying to do production usage?

Reply:
Hi @tcarmelveilleux, Thanks for your review. We need more time to resolve this. Is it ok for us to update this in next pull request?

Reply:
Yes

Reply:
Hi @tcarmelveilleux Thanks for your comments, we will add it in next pull request.